cmd/configure/configure.go

```diff
@@ -0,0 +1,132 @@
+package configure
+
+import (
+	"bytes"
+	_ "embed"
+	"text/template"
+
+	"github.com/Masterminds/sprig/v3"
+)
+
+type device struct {
+	Name            string
+	Title           string
+	Yaml            string
+	ChargerHasMeter bool // only used with chargers to detect if we need to ask for a charge meter
+}
+
+type loadpoint struct {
+	Title       string
```

Comment:
Perspektivisch können wir was aus core wiederverwenden, TODO für später

Reply:
Füge das als kommentar in den code hinzu